src/main/java/segmentation/Colour.java

```diff
@@ -0,0 +1,15 @@
+package segmentation;
+
+import org.joml.Vector3f;
+
+public class Colour {
+
```

Comment:
Add a private (default) constructor to enforce static, singleton pattern (prevent instantiation of Colour class)

Reply:
Added/